spec-main/api-net-spec.ts

```diff
@@ -633,6 +633,142 @@ describe('net module', () => {
       })
     })
 
+    it('should not use the sessions cookie store by default', (done) => {
+      respondOnce.toSingleURL((request, response) => {
+        response.statusCode = 200
+        response.statusMessage = 'OK'
+        response.setHeader('x-cookie', `${request.headers.cookie!}`)
+        response.end()
+      }).then(serverUrl => {
+        const sess = session.fromPartition('cookie-tests-1')
+        const cookieVal = `${Date.now()}`
+        sess.cookies.set({
+          url: serverUrl,
+          name: 'wild_cookie',
+          value: cookieVal
+        }).then(() => {
```

Comment:
in https://github.com/electron/electron/pull/22596 it looks like a flush might be required to ensure that this makes it to the store..? is that test wrong or is this one wrong?

Reply:
This one works 😄 🤷‍♂ Ran it quite a few times locally, flush is not required afaics

Reply:
The test previously was erroneously passing, and with your changes may be coincidentally passing. Can you check if this test still passes when you reverse the flag? It was my experience that the cookies needed to flush for the chromium machinery to pick them up.

Reply:
@NilSet There are two tests here, one for and one against.  They can't both be erroneously passing.

i.e. These tests are testing both the positive and negative cases 👍

Reply:
Cookie flush is needed only if you have a short running app and want to ensure that they get persisted on shutdown. Adding flush here has no negative side effect but it isn't needed.